services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -81,7 +81,7 @@ export const DropdownField = ({
   // update form data
   const onChangeHandler = async (event: InputChangeEvent) => {
     const selectedOption: SelectedOption = {
-      label: event.target.id,
+      label: event.target.name,
```

Comment:
CMSDS updated this so that instead of receiving an ID and Value, you get a name and value.